### DISCUSSION TITLE
Annotations for Real-World Correlations

### DISCUSSION THREAD

**Discussion by metmarkosaric** [plausible/analytics#97] [Anything Else]
"I just switched all my traffic using a redirect from es6katas.org to jskatas.org. Now I would love to see a marker (a straight vertical line) which I can give some text, to understand where possible traffic changes might have come from. I could imagine many use cases for setting such a marker, such as:

    a social media campaign
    an announcement of some sort
    a new feature deploy
    an ad campaign
    etc."

**UPDATE**: Annotations are now live! Hover over the date on the top chart where you want to place the annotation, right-click to open the context menu and select “Add note”. Annotations can be personal or shared with your team. More at https://plausible.io/docs/annotations

**Comment by chrisshennan** [comment]
This would be great. It would allow me to keep track of big changes that might influence traffic and allow others to see what worked / didn't work

**Comment by liamjdillon** [comment]
+1 for this feature. As an online marketer, this would be hugely valuable (especially if it had team commenting / discussion abilities like a Facebook thread with mentions so people could discuss a particular annotated milestone).

**Comment by zanderle** [comment]
This would be awesome!!

**Comment by mathetos** [comment]
+1 for this feature request. Google Analytics calls them "annotations". Here's what that looks like in GA. The 3rd column I've obfuscated, but it's for identifying who made the annotation. 

you can see the "speech bubble" along the timeline where the annotation is placed. When you click on a speech bubble, it's cooresponding annotation is highlighted in the list below. 
![image](https://user-images.githubusercontent.com/930724/150802945-adc3fb10-30bb-4e52-8152-db865a7dbe80.png)

**Reply by zestylemon** [reply]
This is the layout everyone switching from GA will be familiar with, would be happy for this simple format to be adopted by Plausible.

**Comment by SemaphoreOxalis** [comment]
+1 on this ! Plausible is so convenient to use that all my client will be more interested to look at their stats and use a "annotation" function

**Comment by Pryla** [comment]
Great idea!

**Comment by sovstack** [comment]
+1 on this feature. I purchased Plausible before realizing that this feature didn't exist. This is the only feature I miss from Google Analytics since I've been a paying customer of Plausible.

Other sites I manage will not pay for Plausible due to lack of this specific feature. It is significant as it provides context for the marketing and web team.

**Comment by zestylemon** [comment]
+1 - I’m about to switch several websites to Plausible - really hoping this feature is added sooner rather than later. 🤞

**Comment by nowori** [comment]
+1 necessary to provide context, keep track of efforts and their results

(additional keywords for search: comments, notes)

**Comment by winstonwolff** [comment]
Yes our clients also find this small feature to have big value as others have said. Without this, they have to cross-reference another document to see how changes in site traffic relate to marketing efforts, new features, and outside things occurring.

**Comment by blqke** [comment]
This feature is missing in Google Analytics 4, and some Chrome extensions started to appear to resolve this ⤵️
![image](https://user-images.githubusercontent.com/1248148/184538796-698cda48-18e6-4cd8-929a-f2cafa74b0a9.png)

So I guess this is one great pain point to fix, and another reason to get GAnalytics users make the switch 😁

**Comment by benjaminhouy** [comment]
I would love to see this feature as well.

I just found out I made a mistake and one of my pages wasn't displaying properly, leading to very low conversion rates for a related goal. With this feature, I would be able to put a marker on the last 7 days so I can later remember why the conversions for that period were so low.

**Comment by Vigasaurus** [comment]
Recently noticed a pretty nice implementation of this on Google Finance - a horizontally scrollable list of events, as well as the annotations on the graph, both of which in my opinion fit Plausible's design language much better than the GA table format. Personally, I'd also like to be able to make annotations for a range of time, instead of a point in time and be able to easily filter to/swap time period to match said annotation interval. 

This also opens the door for adding better context in annotations (title, description, link (for A/B tests, or ad campaigns etc)), as well as a "Up x% across this range" or a "Down y% since this date" automatic metric. 

Also notice the ability to toggle the Key Events on and off, which entirely hides the items on the graph and the scrollable list, which I imagine would be the least intrusive way to implement something like this. 

![Screenshot_20221111-161400.png](https://user-images.githubusercontent.com/19434157/201438487-9beda5a5-160b-47d4-8d76-c99f9745e34e.png)

**Reply by sovstack** [reply]
Whoa, this is nice. That would be a great implementation to have to Plausible.

**Comment by techroad-dev** [comment]
+1 but I am surprised, it's been 3 years already😡 and this extremely basic feature is still missing from the app.
What's the point of taking features requests when the Dev team won't even consider them.

**Comment by techroad-dev** [comment]
Without this feature, this product is not useful at least for companies who need to perform optimizations to their pages.
I am happy to migrate to plausible but without this feature, highly unlikely.

![image](https://github.com/plausible/analytics/assets/101328318/ba9e5b75-40f3-402d-a40f-b59cb513fe79)

**Comment by ribtoks** [comment]
This is a very important feature. PostHog nailed it in the way it's presented in all charts.

**Comment by Gotterbild** [comment]
Was surprised this simple feature is absent in Plausible. +1 for it since we use it. We keep test-driving Plausible in parallel with other analytics services since we migrated from GA, and absence of this feature could be a show stopper for Plausible.

**Comment by jdorner4** [comment]
Please give this feature a bump at: https://feedback.plausible.io/174

**Comment by sovstack** [comment]
It has been 4 years since this thread was created and still no sort of notation or notes added to the timeline of Plausible. If you are reading this and want this feature too, go here: [Add annotations to specific dates in the top chart](https://feedback.plausible.io/174) and upvote it.

At this point, after years of waiting for this feature on Plausible, the first privacy analytics platform with this feature will have my business. The lack of this feature has been very problematic, making our stats unusable as we cannot add context.

**Comment by metmarkosaric** [comment]
Annotations are now live! Hover over the date on the top chart where you want to place the annotation, right-click to open the context menu and select “Add note”. Annotations can be personal or shared with your team. More at https://plausible.io/docs/annotations

**Reply by ribtoks** [reply]
Not clear from the docs, are they available on self-hosted version?

**Reply by metmarkosaric** [reply]
@ribtoks Annotations were just introduced to the Plausible cloud today. We plan to include them in the next self-hosted release hopefully later this year. Community Edition (self-hosted) is a long-term release, so new features do not appear automatically as they do on Cloud. Once the new CE release is available, you will need to upgrade your CE instance to get the new features.
